kubernetes/structures_pod.go

```diff
@@ -606,6 +639,33 @@ func expandSecretVolumeSource(l []interface{}) *v1.SecretVolumeSource {
 	return obj
 }
 
+func expandTolerations(tolerations []interface{}) []v1.Toleration {
```

Comment:
This method should return a slice of pointers rather then values, like this`[]*v1.Toleration`.

Reply:
The `v1.PodSpec` type expects a slice of literals, same as the `Containers` field, so this is the same as `expandContainers`. Unless I'm missing something? 😃

Reply:
We're trying to standardize on this pattern of expanders returning pointers. The bulk of the change happened in https://github.com/terraform-providers/terraform-provider-kubernetes/pull/231
In doing this, among other things, we benefit from detecting the inadvertent use of nil return values early on during test runs (as might be the case when an error occurred).